01-introduction-and-course-organization/CONSIDERATIONS_FOR_A_DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -0,0 +1,62 @@
+# Considerations for a development environment
+
+## Do not store your projets' sources on a cloud provider
+
+We highly recommend to **never store projects' sources your files on
```

Comment:
maybe : to **never store your projects' sources files on...

Reply:
Thanks! It's fixed.